doc/api_assets/api.js

```diff
@@ -136,6 +136,30 @@
     updateHashes();
   }
 
+  function setupFlavorSelectors() {
+    const kFlavorPreference = 'customFlavor';
+    const vCommonJS = 'cjs';
+    const flavorSetting = sessionStorage.getItem(kFlavorPreference);
+    const flavorSelectors = document.querySelectorAll('.js-flavor-selector');
+
+    flavorSelectors.forEach((selector) => {
+      selector.checked = flavorSetting !== vCommonJS;
+      selector.addEventListener('click', (e) => {
+        const checked = e.target.checked;
+
+        if (checked) {
+          sessionStorage.removeItem(kFlavorPreference);
+        } else {
+          sessionStorage.setItem(kFlavorPreference, vCommonJS);
+        }
+
+        flavorSelectors.forEach((el) => {
+          el.checked = checked;
+        });
+      });
```

Comment:
```suggestion
      selector.addEventListener('change', (e) => {
        const checked = e.target.checked;

        sessionStorage.setItem(kFlavorPreference, checked);

        flavorSelectors.forEach((el) => {
          el.checked = checked;
        });
      });
```

Reply:
Elsewhere in the file it's referred to as "selector", see the "flavorSelector" function in the same file. Would maybe be even more confusing if it's not the same as in the other function. The class name also ends with "selector".

I tried to mimic the style of the other functions to increase coherency.

Reply:
I pushed a small refactor to fix the initial state of the togglers that this suggestion led to.

Reply:
Also keep in mind that it leaves "garbage" sessionStorage values, where my initial solution kept it clean. But I guess that's worth it.

Reply:
Yeah, since it's `sessionStorage`, those "garbage" values are very unlikely to cause problems :+1: